### PR TITLE
Create Markdown extension to parse video tag

### LINKED ISSUE #2: Add Markdown extension to parse video url
The Markdown should be:

```
@[video](url)
```

which should produce the html:

```
<video src="url" />
```

### PR DESCRIPTION
**Pull request by daveraine**
Closes #2

### CODE REVIEW

**Comment by Kitcheone** [comment]
👍

File: src/Piston.Tests/Markdown/VideoUrlTests.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+            Assert.Equal(@"# Heading
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+
+<video src=""http://site.com/video.mp4"" />
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.", result);
+        }
+
```
Comment (Kitcheone) [inline src/Piston.Tests/Markdown/VideoUrlTests.cs:28]: Is it worth adding a test to ensure markdown containing plain text "video" is not accidentally transformed into a <video> element?

**Review by Kitcheone — COMMENTED** [review]
Add extra test!

File: src/Piston.Tests/Markdown/VideoUrlTests.cs (hunk 1 above)
Comment (daveraine) [reply src/Piston.Tests/Markdown/VideoUrlTests.cs:28]: I could do but I know it won't because of the regex 😉

Comment (Kitcheone) [reply src/Piston.Tests/Markdown/VideoUrlTests.cs:28]: Future proof! 😼

Comment (daveraine) [reply src/Piston.Tests/Markdown/VideoUrlTests.cs:28]: Done... happy now? :stuck_out_tongue:

Comment (Kitcheone) [reply src/Piston.Tests/Markdown/VideoUrlTests.cs:28]: Yes 😄

**Pull request merged**